index.html

```diff
@@ -1,56 +1,220 @@
+            <p class="my-profile">
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <a class="contact" href="#contacts">LET'S CONNECT</a>
             <!-- Social media links -->
             <ul class="social-media">
-                <li><a href="https://twitter.com/jerryowusu11">
-                    <i class="fa fa-twitter fa-lg" aria-hidden="true"></i>
-                </a></li>
-                <li><a href="https://linkedin.com/in/jeremiahowusu">
-                    <i class="fa fa-linkedin fa-lg" aria-hidden="true"></i>
-                </a></li>
-                <li><a href="https://medium.com/jerryowusu">
-                    <i class="fa fa-medium fa-lg" aria-hidden="true"></i>
-                </a></li>
                 <li><a href="https://github.com/jerryowusu">
-                    <i class="fa fa-github fa-lg" aria-hidden="true"></i>
-                </a></li>
+                        <i class="fa fa-github fa-lg" aria-hidden="true"></i>
+                    </a></li>
+                <li><a href="https://linkedin.com/in/jeremiahowusu">
+                        <i class="fa fa-linkedin fa-lg" aria-hidden="true"></i>
+                    </a></li>
                 <li><a href="https://angellist.com/jerryowusu">
-                    <i class="fa fa-angellist fa-lg" aria-hidden="true"></i>
-                </a></li>
-            </ul>      
+                        <i class="fa fa-angellist fa-lg" aria-hidden="true"></i>
+                    </a></li>
+                <li><a href="https://twitter.com/jerryowusu11">
+                        <i class="fa fa-twitter fa-lg" aria-hidden="true"></i>
+                    </a></li>
+                <li><a href="https://medium.com/jerryowusu">
+                        <i class="fa fa-medium fa-lg" aria-hidden="true"></i>
+                    </a></li>
+            </ul>
+            <button class="button resume" type="button">Get my resume</button>
+            <!-- Categories -->
+          
+              <ul class="categories">
+                  <li class="tech-languages category">
+                      Languages                 
+                     <i class="fa fa-angle-down fa-lg angle-arrow" aria-hidden="true"></i>                                                              
+                  </li>
+                 <li> <ul class="languages">
+                    <li class="language" >
+                        <img class="language-image" src="images/js.png" alt="JS"> 
+                        JavaScript
+                    </li>
 
+                    <li  class="language" >
+                        <img class="language-image" src="images/HTML.png" alt="HTML">
+                         HTML 
```

Comment:
- [x]  Great job adding languages icons, but unfortunately the image icon for `HTML` language is not showing up in your project as you can see in the screenshots below. Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. 

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/148658619-52cf2257-97c0-4186-9465-b80f638fa24d.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/148658622-a3a4a51e-9e11-4ed2-8f39-8bf03b7f641e.PNG)

Reply:
Thanks a lot for the swift response, @Hamzaoutdoors. I am confused as to why it did not work on your system since it was working fine on mine before pushing. By the way, I have made further changes and I believe it will work fine now. Thanks again.